docs/src/_guide/orchestration.md

```diff
@@ -13,34 +13,37 @@ When a new pipeline schedule is created using the [UI](/reference/ui) or [CLI](/
 
 ## Create a Schedule
 
-### Specifying extractors and loaders
+### Scheduling predefined jobs
 
-To regularly schedule your ELT to run, use the ["Pipelines" interface in the UI](/reference/ui#pipelines), or the [`meltano schedule` command](/reference/command-line-interface#schedule):
+To regularly schedule your pipeline to run first define it as a [job](/reference/command-line-interface#job) within your project.
+Then you can schedule it directly using the `meltano schedule add` command:
 
 ```bash
-meltano schedule add [SCHEDULE_NAME] --extractor [EXTRACTOR_NAME] --loader [TARGET_NAME] --interval [INTERVAL]
-```
-
-Example:
+# Define a job
+meltano job add tap-gitlab-to-target-postgres-with-dbt --tasks "tap-gitlab target-postgres dbt:run"
 
-```bash
-meltano schedule add carbon__sqlite --extractor tap-carbon-intensity --loader target-sqlite --interval="@daily"
+# Schedule the job
+meltano schedule add daily-gitlab-load --job tap-gitlab-to-target-postgres-with-dbt --interval '@daily'
 ```
 
 Now that you've scheduled your first pipeline, you can load the "Pipeline" page in the UI and see it show up.
 
-### Scheduling predefined jobs
+### Specifying extractors and loaders
 
-If you've defined a [job](/reference/command-line-interface#job) within your project, you can schedule it directly without having to manually pass its plugins to the `meltano schedule add` command:
+If you'd prefer not to define a job and just schedule a simple ELT pipeline instead you can pass an Extractor, Loader, and Transform directly via the CLI:
 
+```bash
+meltano schedule add [SCHEDULE_NAME] --extractor [EXTRACTOR_NAME] --loader [TARGET_NAME] --interval [INTERVAL]
 ```
-# Define a job
-meltano job add tap-gitlab-to-target-postgres-with-dbt --tasks "[tap-gitlab target-postgres, dbt:run]"
 
-# Schedule the job
-meltano schedule add daily-gitlab-load --job tap-gitlab-to-target-postgres-with-dbt --interval '@daily'
+Example:
+
+```bash
+meltano schedule add carbon__sqlite --extractor tap-carbon-intensity --loader target-sqlite --interval="@daily"
 ```
 
+You can also schedule pipelines using the ["Pipelines" interface in the UI](/reference/ui#pipelines).
```

Comment:
We may want to note that the UI only supports scheduling legacy ELT tasks. It doesn't support jobs.

Reply:
Same as above--maybe we just drop?

Reply:
I'd vote for that, not sure how other folks would feel about that though.

Reply:
CC @DouweM @tayloramurphy in case y'all have opinions on dropping this.

Reply:
We should keep existing functionality in the UI working, but don't need to add any other. So if using the UI for orchestration is no longer actually a good path to set users on, we should remove the mention.

Reply:
@DouweM @alexmarple I'll go ahead and leave this mention of the UI out of the docs for now then.